cmd/utils/flags.go

```diff
@@ -1920,6 +1925,12 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 	if ctx.IsSet(EnableTrustProtocolFlag.Name) {
 		cfg.EnableTrustProtocol = ctx.IsSet(EnableTrustProtocolFlag.Name)
 	}
+	if ctx.IsSet(DisableBscProtocolFlag.Name) {
+		if ctx.Bool(MiningEnabledFlag.Name) {
```

Comment:
check `cfg.DisableBscProtocol` is better after DisableBscProtocol is assigned
otherwise if `--disablebscprotocol false`
here will result wrong behavior